### PR TITLE
chore: Add pre-commit instructions to issue template

### PR DESCRIPTION
**Pull request by Techassi**
This adds instructions to update dependencies related to the pre-commit workflows.

<details>
<summary>Issue Template Preview at 0158cbf</summary>

---

## Pre-Release Updates for Stackable Release XX.(X)X

Part of <https://github.com/stackabletech/issues/TRACKING_ISSUE>

### Update pre-commit Workflow

> [!NOTE]
> The pre-commit config and workflows need to be kept up-to-date to ensure
> usage of recent tooling versions. This requires some manual work in this
> repository.

```[tasklist]
### Tasks
- [ ] Update `python-version` in local and templated `pr_pre-commit.yml` workflow
- [ ] Update hook refs in local and templated `.pre-commit-config.yaml` file
- [ ] Update Hadolint version in the `config/versions.yaml` file
```

### Update Rust Toolchain

> [!NOTE]
> During a Stackable release we need to ensure that every product operator uses
> the latest Rust toolchain (used by us). To keep the toolchain in sync across
> all our operators, we update the version centrally in this repository.

```[tasklist]
### Tasks in this Repository
- [ ] Update Rust toolchain in the `config/versions.yaml` file.
- [ ] Generate downstream PRs using the ["Generate Downstream PRs"](https://github.com/stackabletech/operator-templating/actions/workflows/generate_prs.yml) action.
- [ ] Merge downstream PRs, see below for more details.
```

Replace the items in the task lists below with the applicable Pull Requests

```[tasklist]
### Tasks in Downstream Operator Repositories
- [ ] https://github.com/stackabletech/airflow-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/commons-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/druid-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/edc-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/hbase-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/hdfs-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/hello-world-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/hive-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/kafka-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/listener-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/nifi-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/opa-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/secret-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/spark-k8s-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/superset-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/trino-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files
- [ ] https://github.com/stackabletech/zookeeper-operator/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files

```

</details>

### CODE REVIEW

**Review by razvan — DISMISSED** [review]
nice

**Review by lfrancke — DISMISSED** [review]

File: config/versions.yaml [hunk 1]
```diff
@@ -3,4 +3,8 @@
 # If you change the Rust toolchain version here, make sure to also change
 # docker-images/ubi8-rust-builder/Dockerfile & docker-images/ubi9-rust-builder/Dockerfile
 rust_version: 1.80.0
```
[the remark this answers is not in the document]
Comment (lfrancke) [reply config/versions.yaml:5]: I also rolled this update into my #430 PR if you want to ignore this

**Comment by Techassi** [comment]
I just realized that I removed Hadolint, which is still helpful to have because we template the operator Dockerfile. See b850bc6 which partly reverts 8b3033d.

File: config/versions.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Techassi) [reply config/versions.yaml]: I added it in d3fcbb2

**Review by lfrancke — APPROVED** [review]

**Pull request merged**